<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsMenu</class>
 <widget class="QDialog" name="StatisticsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="session_stats_tab">
      <attribute name="title">
       <string>Session Stats</string>
      </attribute>
      <widget class="QGroupBox" name="scan_count_box">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>371</width>
         <height>69</height>
        </rect>
       </property>
       <property name="title">
        <string>Session Scan Count</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="scan_count_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="category_count_box">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>371</width>
         <height>158</height>
        </rect>
       </property>
       <property name="title">
        <string>Categorical Scan Count</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="glass_lvl">
          <property name="text">
           <string>Glass</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="plastic_progressbar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="aluminium_lbl">
          <property name="text">
           <string>Aluminium</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="aluminium_progressbar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QProgressBar" name="glass_progressbar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="plastic_lbl">
          <property name="text">
           <string>Plastic</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="other_lbl">
          <property name="text">
           <string>Other</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QProgressBar" name="other_progressbar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="overall_stats_tab">
      <attribute name="title">
       <string>Overall Stats</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>271</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For overall stats across all sessions, click the button to go to the dashboard.&lt;/p&gt;&lt;p&gt;(Requires database to be enabled)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Go to Dashboard</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
